addon/src/lib/addonManager.js

```diff
@@ -1,66 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
now using the `management` api to fetch addons and determine which txp addons are installed at the time of the daily ping.